src/play_cljc/gl/utils.cljc

```diff
@@ -18,15 +18,15 @@
     (gl game attachShader program vertex-shader)
     (gl game attachShader program fragment-shader)
     (gl game linkProgram program)
-    (if #?(:clj (= (gl game TRUE) (gl game getProgrami program (gl game LINK_STATUS))) 
+    (if #?(:clj (= (gl game TRUE) (gl game getProgrami program (gl game LINK_STATUS)))
            :cljs (gl game getProgramParameter program (gl game LINK_STATUS)))
       program
       (throw (ex-info (gl game getProgramInfoLog program) {})))))
 
 (defn create-buffer
   ([game program attrib-name src-data]
    (create-buffer game program attrib-name src-data {}))
-  ([game program attrib-name src-data
+  ([game ^Integer program ^String attrib-name src-data
```

Comment:
`Integer` and `String` don't exist in cljs, I wonder if it just ignores them? I think these are unnecessary anyway because create-buffer only runs on initialization. Probably best to not include them.

Reply:
Oh, yeah that is weird. I'm not too familiar with clojurescript, so I don't know what it's doing with those. I'll remove them.